src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
 public class WorkWithFile {
+    private static final String BUY_WORD = "buy";
+    private static final String SUPPLY_WORD = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToTheFile(toFileName, createReport(readDataFromFile(fromFileName)));
+    }
+
+    private String[] readDataFromFile(String nameOfFile) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(nameOfFile))) {
+            String temporary;
+            while ((temporary = bufferedReader.readLine()) != null) {
+                stringBuilder.append(temporary).append(" ");
```

Comment:
```suggestion
                stringBuilder.append(temporary).append(" ");
```
" "-  you use it several times so better create constant  - WHITESPACE_ SEPARATOR

Reply:
don't fixed